diff --git a/node_modules/hapi-plugin-websocket/hapi-plugin-websocket.d.ts b/node_modules/hapi-plugin-websocket/hapi-plugin-websocket.d.ts
index ce49b8e..a677685 100644
--- a/node_modules/hapi-plugin-websocket/hapi-plugin-websocket.d.ts
+++ b/node_modules/hapi-plugin-websocket/hapi-plugin-websocket.d.ts
@@ -23,7 +23,7 @@
 */

 import { Plugin, Request, ReqRef, ReqRefDefaults } from "@hapi/hapi"
-import { Podium }                                  from "@hapi/podium"
+import Podium                                      from "@hapi/podium"
 import * as ws                                     from "ws"
 import * as http                                   from "node:http"

